test/automation/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-automation",
-  "version": "1.39.0",
```

Comment:
why are you changing this?

Reply:
nope will revert. Something in my dev environment always causes this inadvertent edit. I have not figured out as to what does this.

Reply:
reverted.